lib/checkclass.cpp

```diff
@@ -2383,6 +2383,12 @@ void CheckClass::duplInheritedMembersError(const Token *tok1, const Token* tok2,
 // Check that copy constructor and operator defined together
 //---------------------------------------------------------------------------
 
+enum CtorType {
+    NO = 0,
```

Comment:
I believe these explicit values can be removed if review comments below are followed